test/rocarray/base.jl

```diff
@@ -27,6 +27,31 @@ end
     @test collect(c) == 3:4
 end
 
+@testset "resize!" begin
+    a_h = Array(range(1, 10))
+    a_d = a_h |> roc
+    # Resize up
+    resize!(a_h, 15)
+    resize!(a_d, 15)
+    a_h[10:15] .= 15
+    a_d[10:15] .= 15
+    @allowscalar begin
+        @test a_h == a_d
```

Comment:
This won't work due to uninitialized data, you can just remove this test.

Reply:
See above comment. I initialize the new elements to have the same value on both host and device. Therefore the equality should pass.